src/main/resources/META-INF/rewrite/apache-httpclient-5.yml

```
@@ -487,6 +487,9 @@ recipeList:
   - org.openrewrite.java.ChangeMethodName:
       methodPattern: org.apache.http.HttpMessage getAllHeaders()
       newMethodName: getHeaders
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.apache.hc.client5.http.classic.methods.HttpPost getURI()
+      newMethodName: getUri
```

I've briefly tried the following, but no luck just yet.
```suggestion
      methodPattern: org.apache.hc.client5.http.classic.methods.HttpUriRequest getURI()
      newMethodName: getUri
      matchOverrides: true
```